packages/meta-viewer/src/lib/meta-viewer.ts

```diff
@@ -0,0 +1,63 @@
+import {Address, Anatomy, MAX_WEIGHT, ProductType} from '@phuture/types';
+import { SavingsVault } from '@phuture/savings-vault';
+import { Index } from '@phuture/index';
+import { PhutureError } from '@phuture/errors';
+import {Erc20Permit} from "@phuture/erc-20";
+
+export class MetaViewer {
```

Comment:
I don't think we would go with an approach of additional Viewer class, but instead, what we need to do is to create a TS interface PhutureProduct (ERC20-ish with `constituents` method), which Index and SV would implement

Reply:
It can be done like that so the client doesn't need to construct another class.